bolt12/format-string-test.json

```diff
@@ -0,0 +1,47 @@
+[
+    {
+	"comment": "A complete string is valid",
+	"valid": true,
+	"string": "lno1qcp4256ypqpq86q2pucnq42ngssx2an9wfujqerp0y2pqun4wd68jtn00fkxzcnn9ehhyec6qgqsz83qfwdpl28qqmc78ymlvhmxcsywdk5wrjnj36jryg488qwlrnzyjczlqsp9nyu4phcg6dqhlhzgxagfu7zh3d9re0sqp9ts2yfugvnnm9gxkcnnnkdpa084a6t520h5zhkxsdnghvpukvd43lastpwuh73k29qsy"
```

Comment:
Looks like the test vectors contain fields that have been removed from the BOLT. Do you have any updated ones that I can use for testing?

Reply:
Also running into issues with the first four test vectors here, specifically even type 28 which isnt't in the spec anymore. 

Also ran into an ordering issue (TLV 28 is followed by 19?), but that could be a bug my side.

Reply:
Hmmm... I was able to parse it but the TLV had type 26 (not 28). Not sure about 19, but AFAICT all removed TLVs had even types.

Reply:
OK, putting "regenerate test vectors" on my TODO now...